samples/rx65ncloudkit/iotc-azrtos-sdk/.cproject

```diff
@@ -175,7 +176,7 @@
 						</toolChain>
 					</folderInfo>
 					<sourceEntries>
-						<entry excluding="libTO/examples|libTO/src/wrapper|authentication/libto/examples|authentication/libto/src/aes-gcm-sw|authentication/libto/src/wrapper|hardware/libto/src/aes-gcm-sw|hardware/libto/doc|hardware/libto/examples|hardware/libto/src/wrapper" flags="VALUE_WORKSPACE_PATH|RESOLVED" kind="sourcePath" name=""/>
+						<entry excluding="authentication/libto/src/aes-gcm-sw|authentication/libto/src/|hardware/libto/src/aes-gcm-sw|hardware/libto/doc|hardware/libto/examples|hardware/libto/src/|deps/libTO/Sources/examples/|deps/libTO/Sources/src/wrapper/|deps/cJSON/tests/" flags="VALUE_WORKSPACE_PATH" kind="sourcePath" name=""/>
```

Comment:
As of latest version on main, you should be able to compile it without libTO, so it should be possible to remove all of it. I think that the project will compile  - we should have removed all sdk-level dependencies and it should only be needed by projects which use libTO or TO136 secure element along with to_auth_driver.*. 

This dependency removal is optional though, so feel free to treat this as a suggestion. We could revisit this at a later time.